content/_partials/toptoolbar.html.haml

```diff
@@ -1 +1,2 @@
-%jio-navbar#ji-toolbar.fixed-nav{:property=>"https://www.jenkins.io", :showSearchBox=>""}
+- showSearchBox = !page.hide_search_bar && true
```

Comment:
What does the && true do?

Reply:
Yeah! the code snippet is equivalent to `!page.hide_search_bar`.  Initially, for better clarity, I employed `!page.hide_search_bar && true` for my understanding. Let me change it. Thanks for poiniting it out @halkeye